.github/workflows/build.yml

```diff
@@ -21,7 +21,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sdk: [stable]
+        # TODO(srujzs): Replace with stable when 3.3 is out.
+        sdk: [beta]
```

Comment:
Can you elaborate what you mean? Wouldn't any branch testing stable come across analysis issues because the experiment flag is not enabled yet?

Reply:
I was thinking two branches at different states. Similar to our SDK main/dev/beta/stable, we could have web-main and web-published. The web-published branch would test the code at the current published version of the package. Then we could detect if an SDK changes breaks it.

Today (and this is true for most packages, I believe), we only test the unpublished code at `main`, we may not be aware of issues concerning current users of the package.

I sense this is the wrong CI to test that, though. Maybe a better approach would be to create an "examples" package that uses the stable SDK and regular `dart pub get` to get the current version `package:web` and runs some tests on top of it.

Reply:
IIUC, this is kind of another dimension to the matrix where we're testing not only multiple SDKs but multiple package versions? I like it - it'll let us know if we need to mark a previous package as broken, but I'm not sure if the tooling here is setup for that.

> Maybe a better approach would be to create an "examples" package that uses the stable SDK and regular dart pub get to get the current version package:web and runs some tests on top of it.

That would work as a workaround. It'd be nice to have a package test multiple versions of a dependency in this file as well. This would have been useful when we were dealing with the `>=0.3.0 <0.5.0` constraints Flutter packages had to use.